code/AssetLib/AMF/AMFImporter_Postprocess.cpp

```diff
@@ -329,8 +329,8 @@ void AMFImporter::Postprocess_AddMetadata(const AMFMetaDataArray &metadataList,
     sceneNode.mMetaData = aiMetadata::Alloc(static_cast<unsigned int>(metadataList.size()));
     size_t meta_idx(0);
 
-    for (const AMFMetadata &metadata : metadataList) {
-        sceneNode.mMetaData->Set(static_cast<unsigned int>(meta_idx++), metadata.Type, aiString(metadata.Value));
+    for (const AMFMetadata *metadata : metadataList) {
+        sceneNode.mMetaData->Set(static_cast<unsigned int>(meta_idx++), metadata->Type, aiString(metadata->Value));
```

Comment:
error: loop variable ‘metadata’ of type ‘const AMFMetadata&’ binds to a temporary constructed from type ‘AMFMetadata* const’ [-Werror=range-loop-construct]
  332 |     for (const AMFMetadata &metadata : metadataList) {